### PR TITLE
Added Readme file for API Tests

### PR DESCRIPTION
**Pull request by rishagg01**
# Added Readme file for API tests #

## 🗣 Description ##

<The Readme file is located on the following folder: LME/testing/tests>

## 💭 Motivation and context ##

<Follow the Readme to be able to run API tests automatically on LS1>

## 🧪 Testing ##

<Performed Preview and ran through the steps before checking it in>

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.

## ✅ Post-merge checklist ##
Fixes 173

- [ ] Create a release.

### CODE REVIEW

**Review by llwaterhouse — APPROVED** [review]

File: testing/tests/Readme.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Checkout the following LME Branch on your Azure LME folder: cbaxley-122-testbed_from_scripts
+After the Git Pull has been completed, go to LME/testing folder and run the following command:
+```
+./SetupTestbed.ps1 -NumClients 2 -AutoShutdownTime 0000 -AllowedSources "<YourIPAddress>" -l <Location> -ResourceGroup <ResourceGroupName> -m  -y
+```
+Note: The -m option is for minimum install, will only install the Linux Server. If you want a full install, leave out -m from the above command.
+
+After the above command has been completed, run the following command:
+```
+./InstallTestbed.ps1 -ResourceGroup <ResourceGroupName> -m -v 1.3.2  | Tee-Object -FilePath "./L1.output.log"
```
Comment (cbaxley) [inline testing/tests/Readme.md:13]: Change `L1.output.log` to `<ResourceGroupName>.output.log`

File: testing/tests/Readme.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Installation steps to run API tests
+
+Run through the following steps to get API tests up and running on your Azure test bed.
+Checkout the following LME Branch on your Azure LME folder: cbaxley-122-testbed_from_scripts
```
Comment (cbaxley) [inline testing/tests/Readme.md:4]: We will have to change this to "main" when we merge the168 branch.

**Review by cbaxley — CHANGES_REQUESTED** [review]

File: testing/tests/Readme.md (hunk 2 above)
Comment (llwaterhouse) [reply testing/tests/Readme.md:4]: How long will it be before you think you'll be merging the 168 branch?  Is it worth changing now?

Comment (cbaxley) [reply testing/tests/Readme.md:4]: Trying to merge it tomorrow. Should be merged before this ticket is merged though.

Comment (cbaxley) [reply testing/tests/Readme.md:4]: Well, before the branch this pr is merging into.

**Review by cbaxley — APPROVED** [review]

**Review by llwaterhouse — APPROVED** [review]

**Comment by rishagg01** [comment]
This PR is now obsolete since main already has a read me that covers the information needed to run API tests

**Pull request closed without merging**